openedx/core/lib/request_utils.py

```diff
@@ -118,9 +118,9 @@ def process_request(self, request):
             for name, value in request.COOKIES.items()
         }
         for name, size in cookie_names_to_size.items():
-            metric_name = 'cookies.{}.size'.format(name)
-            newrelic.agent.add_custom_parameter(metric_name, size)
-            log.debug(u'%s = %d', metric_name, size)
+            attribute_name = 'cookies.{}.size'.format(name)
+            newrelic.agent.add_custom_parameter(attribute_name, size)
```

Comment:
We shouldn't have any `newrelic.agent` calls.  We should be calling the wrapped methods from edx-django-utils.  We don't have to fix all, but we can fix the calls in this method while we are changing it.